src/benchmarks/micro/coreclr/PacketTracer/Camera.cs

```diff
@@ -0,0 +1,38 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.Intrinsics.X86;
+using static System.Runtime.Intrinsics.X86.Avx;
+using System.Runtime.Intrinsics;
+
+namespace PacketTracer
+{
+    internal class Camera
+    {
+        public Camera(VectorPacket256 pos, VectorPacket256 forward, VectorPacket256 up, VectorPacket256 right)
+        {
+            Pos = pos;
+            Forward = forward;
+            Up = up;
+            Right = right;
+        }
+
+        public VectorPacket256 Pos;
+        public VectorPacket256 Forward;
+        public VectorPacket256 Up;
+        public VectorPacket256 Right;
+
+        public static Camera Create(VectorPacket256 pos, VectorPacket256 lookAt)
+        {
+            VectorPacket256 forward = (lookAt - pos).Normalize();
+            VectorPacket256 down =
+                new VectorPacket256(Vector256<float>.Zero, Vector256.Create(-1.0f), Vector256<float>.Zero);
```

Comment:
>new VectorPacket256(Vector256<float>.Zero, Vector256.Create(-1.0f), Vector256<float>.Zero) [](start = 16, length = 90)

Should this be a readonly/const variable created/allocated once?

Reply:
I don't know, I just ported it and made sure it does what expected. I did not have time to analyze all the code